src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/fullScript_approved.properties

```diff
@@ -0,0 +1,59 @@
+# approved scripts
+metadataGatheringDisabled=Metadata gathering disabled. \
+  <p>You can re-enable it permanently by removing the system property: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.metadataGathering</code> </p>\
+  <p>or temporarily (until restart) by running this in the Script Console: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.METADATA_GATHERING = true</code></p>
+
+## table headers
+approvedExpandCode=Code
+approvedLanguage=Language
+usageCount=# of uses
+usageCount_tooltip=Uses since the introduction of metadata
+lastTimeUsed=Date of last use
+wasPreapproved_header_tooltip=The presence of a green tick means the script was approved during the configuration. \
+  Otherwise it was created by a user without the permission to run scripts and was then approved using this page.
+lastApprovalTime=Date of the last approval
+lastApprovalTime_tooltip=A single script can be approved multiple times. It occurs every time the related configuration is saved.
+lastKnownApproverLogin=Last approver
+
+## table content
+noRequesterSinceMetadata=N/A
+noRequesterSinceMetadata_tooltip=The requester was not recorded. \
+  It could be because the request was created before the metadata was introduced or the code calling this script did not provide user information.
+noRequestApprovalTimeSinceMetadata=N/A
+noRequestApprovalTimeSinceMetadata_tooltip=It was requested before the introduction of metadata.
+noContextItemSinceMetadata=N/A
+noContextItemSinceMetadata_tooltip=The context item was not provided by the code asking for approval.
+
+emptyMetadata=There is no metadata for this approval
```

Comment:
```suggestion
emptyMetadata=This script has been approved before Script Security 1.75 and no additional metadata has been recorded yet. Once this script is executed, additional metadata will be added.
```

Reply:
```suggestion
emptyMetadata=This script was approved before Script Security 1.75 and no additional metadata has been recorded yet. Once this script is executed, additional metadata will be added.
```

Reply:
Both proposals seem to pollute the space due to the size of the sentence.

<details>

<summary>Proof</summary>

### After
![Screenshot_2020-09-02_083609_001](https://user-images.githubusercontent.com/2662497/91940244-727b9380-ecf7-11ea-9eb6-9452819e0b0e.png)

### Before (with the icon for tooltip discoverability)
![Screenshot_2020-09-02_083719_001](https://user-images.githubusercontent.com/2662497/91940329-9212bc00-ecf7-11ea-820e-69f83570ec64.png)

### Before (with just the cursor: help)
![Screenshot_2020-09-02_084251_001](https://user-images.githubusercontent.com/2662497/91940821-63e1ac00-ecf8-11ea-96e3-df22ec87b216.png)

</details>

Going with no modification. This could be improved in a follow up PR.